plugins/services/src/js/components/modals/ServiceRestartModal.js

```diff
@@ -79,38 +80,64 @@ class ServiceRestartModal extends React.Component {
     );
   }
 
-  render() {
-    const { isPending, onClose, open, service, restartService } = this.props;
+  getServiceLabel() {
```

Comment:
It looks like this is used in all three cases and is the same code. Would it make sense to add it somewhere centrally?